### ISSUE TITLE
Formatting does not respect <file> like the validator does

### ISSUE THREAD

**Issue by wongjn** [wongjn/vscode-php-sniffer#33]
**Describe the bug**
Formatter from the extension will format files even if they are ignored from validation by `phpcs.xml` or equivalent config file.

**To Reproduce**
1. Create `.phpcs.xml` with:  
```xml
<?xml version="1.0"?>
<ruleset name="Test ruleset">
  <file>index.php</file>
  <rule ref="Generic.PHP.LowerCaseConstant"/>
</ruleset>
```
2. Set extension settings to use whatever binary available, with standard set to `.phpcs.xml`
3. Create `ignore-me.php` with:  
```php
<?php
$foo = TRUE;
```
4. Invoke the VSCode command _Format Document_
5. Observe that the `TRUE` has been formatted to lowercase.

**Expected behavior**
File should remain unchanged.

**Environment**

Environment irrelevant.

**Additional context**

Related to — but not the cause of — #32
